packages/language-server/src/server.ts

```diff
@@ -211,3 +250,26 @@ connection.listen();
 function isXMLView(uri: string): boolean {
   return /(view|fragment)\.xml$/.test(uri);
 }
+
+function executeCommand(
+  textDocument: TextDocument,
+  params: ExecuteCommandParams
+): void {
+  switch (params.command) {
+    case QUICK_FIX_STABLE_ID_COMMAND: {
+      const change = executeQuickFixStableIdCommand({
+        textDocument,
+        // @ts-expect-error - we already checked arguments exist
+        quickFixRange: params.arguments[1],
+        // @ts-expect-error - we already checked arguments exist
+        quickFixIDSuggestion: params.arguments[2],
+      });
+      connection.workspace.applyEdit({
```

Comment:
we discussed online that the applyEdit may need to happen inside `executeQuickFixStableIdCommand`

Reply:
as discussed, because we don't want to pass the connection to another `executeQuickFixStableIdCommand` we'll keep it this way